R/testing.R

```diff
@@ -0,0 +1,167 @@
+#' Create a kelp test harness object.
+#'
+#' Exposes same interface as [kelp()] but with mock responses, can be used
```

Comment:
You might expand a bit on "mock" responses here. I'd forgotten about this and had forgotten.

I wonder if `kelp_fake` might be better than `kelp_harness` as a name too?

Reply:
Added some more detail to this and updated the name.